README.md

```diff
@@ -95,10 +78,27 @@ Files with extension `.gz` are uncompressed automatically
 [1]: docs/TAB_FORMAT.md
 
 ## Options
-DNAconvert uses two parsers for NEXUS format: internal and the one from python-nexus package.
+DNAconvert uses two parsers for NEXUS format: internal (default) and the one from python-nexus package.
 
-In the file `data/cfg.tab` the string of form
+In the file `data/cfg.tab` (found at the installed location) the string of form
```

Comment:
I would like to change this to `DNAconvert/config.json` in ${XDG_CONFIG_HOME} or %APPDATA%. Should I add a commit to this pull request?

Reply:
I was also worried about this, please go ahead!

Reply:
I added the configuration handling, I think this can be merged now